pom.xml

```diff
@@ -153,6 +152,34 @@
   </profiles>
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <version>3.2.4</version>
+        <executions>
+          <execution>
+            <phase>package</phase>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <configuration>
```

Comment:
@andreoss WDYT add `<minimizeJar>true</minimizeJar>` configuration here? Check if the ubber-jar minimize its size. You can see more details [here](https://stackoverflow.com/questions/8817257/minimize-an-uber-jar-correctly-using-shade-plugin).

Reply:
@fabriciofx Added